src/schedule/zephyr_domain.c

```diff
@@ -84,7 +84,7 @@ static void zephyr_domain_thread_fn(void *p1, void *p2, void *p3)
 
 		if (++runs == 1 << CYCLES_WINDOW_SIZE) {
 			cycles_sum >>= CYCLES_WINDOW_SIZE;
-			tr_info(&ll_tr, "ll timer avg %u, max %u, overruns %u",
+			tr_info(&ll_tr, "perf_cycle ll timer avg %u, max %u, overruns %u",
```

Comment:
I think here I'd prefer to keep old naming unless there's a really good reason to change. We might have downstream users with scripts parsing this data, and this would be causing churn for them.

Reply:
Please be consistent with keyword-value pairs. Use everywhere comma as separator or just blanks to help make automatic parsing scripts for trace log. I'd also align the previous perf traces to same format.